src/main/java/Todo.java

```diff
@@ -0,0 +1,14 @@
+public class Todo extends Task {
+
+    protected String by;
+
+    public Todo(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString() + " (by: " + by + ")";
```

Comment:
Also the same as before

Reply:
Same as what now? Please clarify in place